app/src/main/java/com/github/se/signify/MainActivity.kt

```diff
@@ -77,6 +77,8 @@ class MainActivity : ComponentActivity() {
     val savedTheme = sharedPreferencesTheme.getBoolean(prefKeyIsDarkTheme, false)
     val savedLanguage = sharedPreferencesLanguage.getBoolean(prefKeyLanguage, false)
 
+    updateLanguage(this, if (savedLanguage) "fr" else "en")
```

Comment:
It's sad we're still handling languages as strings and a boolean rather than an enum. Hopefully we'll have time to refactor it someday.